### ISSUE TITLE
Inability to copy morph to the clipboard in Safari

### ISSUE THREAD

**Issue by JensLincke** [LivelyKernel/LivelyKernel#294]
[Visit issue world](http://lively-web.org/issues/14-10-25_Inability_to_copy_morph_to_the_clipboard_in_Safari.html)

reported by Dan:I have had it explained to me that this is "just plain difficult", and that's OK.  But couldn't we have a morph-menu item "copy to clipboard" that would do the job?

**Comment by JensLincke** [comment]
Accessing the clipboard is only allowed when the user triggered it with intent. (e.g. a web site should not be allowed to interfere with a password the user copied)

**Comment by bertfreudenberg** [comment]
Unlike issue #165, which is about copying stuff (like text) to the system clipboard, copying morphs between worlds does not have to involve the system clipboard.

This could be implemented using localStorage/indexedDB (which is shared between tabs in a browser), or via a clipboard server.

**Comment by mroeder** [comment]
I think copying it via clipboard is better then localStorage or the server!
I have used that feature to copy things between Lively installations (that did not even have L2L running). It works so great that using something other than the clipboard would actually restrict possible usages.

**Comment by rksm** [comment]
I agree with Marko. The tool that people use these days is [ZeroClipboard](http://zeroclipboard.org/). Still flash based but nicely packaged. We hadn't this back then ^^

I think it should be fairly easy to integrate. Any volunteers?

**Comment by bertfreudenberg** [comment]
Of course if we can make the clipboard API work on all browsers, then that's the best solution. I'm not saying we should replace that. But if it does not work, it is really limiting, so making it work in more cases is a Good Idea, right?
I mean, if you can fix #165 then this should also resolve this issue here, I suppose.
